tests/cypress/support/commands.ts

```diff
@@ -78,7 +78,7 @@ Cypress.Commands.add('importYaml', ({ clusterName, yamlFilePath }) => {
 
 // Command add and edit Fleet Git Repository
 // TODO: Rename this command name to 'addEditFleetGitRepo'
-Cypress.Commands.add('addFleetGitRepo', ({ repoName, repoUrl, branch, path, gitOrHelmAuth, gitAuthType, userOrPublicKey, pwdOrPrivateKey, keepResources, correctDrift, fleetNamespace='fleet-local', editConfig=false, helmUrlRegex }) => {
+Cypress.Commands.add('addFleetGitRepo', ({ repoName, repoUrl, branch, path, gitOrHelmAuth, gitAuthType, userOrPublicKey, pwdOrPrivateKey, keepResources, correctDrift, fleetNamespace='fleet-local', editConfig=false, helmUrlRegex, deployToTarget }) => {
```

Comment:
don't quote me here but,  if not mistaken, because we are using `deployToTarget` as on/offf switch on an `if` statement. A similiar thing happens with `editConfig` which is not present either in `e2e.ts`. 
Personally, I agree the both should be.
Also, think that the `e2e.ts` is a document where the allowed types of a given parameter is defined, so it helps us also to ensure we don't pass a string if a number is expected.

Reply:
Sure, I will add both entries there in `e2e.ts` file. Also, I will take care for future test cases too.

Reply:
Updated e2e.ts file with appropriate parameters.